File: app/src/main/java/com/arashabd/coffeecraftapp/room/entity/GeneralInfo.java
package com.arashabd.coffeecraftapp.room.entity;


import androidx.room.ColumnInfo;
import androidx.room.Entity;

@Entity
public class GeneralInfo {

    @ColumnInfo(name = "imageURL")
    private String imageUrl;

    @ColumnInfo(name = "detailsImageURL")
    private String detailsImageURL;

    @ColumnInfo(name = "EQDescription")
    private String EQDescription;


}

File: app/src/main/java/com/arash/coffeecraftapp/models/GeneralInfo.kt
package com.arash.coffeecraftapp.models

import com.google.gson.annotations.SerializedName


data class GeneralInfo (

    @SerializedName("imageURL"        ) var imageURL        : String? = null,
    @SerializedName("detailsImageURL" ) var detailsImageURL : String? = null,
    @SerializedName("EQDescription"   ) var EQDescription   : String? = null

)
